next.config.js

```diff
@@ -29,8 +29,17 @@ const nextConfig = {
         destination: `${process.env.NEXT_PUBLIC_BACK_URL}/users/sign_out`,
       },
       {
-        source: '/api-docs/:path*',
-        destination: `${process.env.NEXT_PUBLIC_BACK_URL}/api-docs/:path*`,
+        source: '/users/:path*',
```

Comment:
This rule overwrite all the other rules that points to the `/users/*` paths, so it looks like the rules pointing to `/users/sign_in`, `/users/sign_up`, `/users/sign_out` and `/users/auth/google_oauth2/callback` are redundant and should be removed for brevity.